lib/components/split-pane.js

```diff
@@ -123,7 +142,6 @@ export default class SplitPane extends Component {
         display: 'flex',
         flex: 1,
         outline: 'none',
-        position: 'relative',
```

Comment:
I removed it so each nested maximized pane is able to fill whole tab's space (not only parent group container). I could do this using `position: fixed` for maximized pane, but it overlaps tab labels.

NB! I'm not sure that it doesn't break anything. It seems that it works fine, but might be some edge cases, that I'm not aware of.